### PR TITLE
GCC 10.3

### PR DESCRIPTION
**Pull request by yellowhat**

### CODE REVIEW

**Comment by yellowhat** [comment]
Not sure why "linux tests / unittests (3.9, original) (pull_request)" failed

File: var/spack/repos/builtin/packages/gcc/package.py [hunk 1]
```diff
@@ -29,6 +29,7 @@ class Gcc(AutotoolsPackage, GNUMirrorPackage):
 
     version('master', branch='master')
 
+    version('10.3.0', sha256='64f404c1a650f27fc33da242e1f2df54952e3963a49e06e73f6940f3223ac344')
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/gcc/package.py:32]: Confirmed sha256.

**Review by michaelkuhn — DISMISSED** [review]
LGTM (for the gcc part)

@alalazo How should we handle the archspec part? I saw that you have a similar patch at https://github.com/archspec/archspec-json/pull/29

**Comment by alalazo** [comment]
@michaelkuhn I would keep here only the version bump, and fix the entry in the archspec project. If we can get https://github.com/archspec/archspec/pull/46 and https://github.com/archspec/archspec-json/pull/29 in quickly then we can also update Spack.

**Comment by yellowhat** [comment]
Should I create a new pull request for gcc bump only?

**Comment by alalazo** [comment]
@yellowhat Just remove the change on externals here and ping me.

**Comment by yellowhat** [comment]
@alalazo done.

**Review by alalazo — APPROVED** [review]

**Pull request merged**